src/coreclr/jit/importercalls.cpp

```diff
@@ -7531,16 +7531,18 @@ Compiler::GDVProbeType Compiler::compClassifyGDVProbeType(GenTreeCall* call)
 //
 // Arguments:
 //     methodHnd -- handle for the special intrinsic method
+//     pIsExact  -- returns whether the type is exact
 //
 // Returns:
 //     Exact class handle returned by the intrinsic call, if known.
 //     Nullptr if not known, or not likely to lead to beneficial optimization.
-CORINFO_CLASS_HANDLE Compiler::impGetSpecialIntrinsicExactReturnType(GenTreeCall* call)
+CORINFO_CLASS_HANDLE Compiler::impGetSpecialIntrinsicExactReturnType(GenTreeCall* call, bool* pIsExact)
```

Comment:
The comment above says that the return value is exact class handle. We should not need the `pIsExact` argument - it is going to be true iff the return value is non-null.

Reply:
Yeah this is an artifact of the fact that I originally thought I wouldn't be able to return exact handles in some cases, but in the end it turned out to not be true.